crates/uv-publish/src/lib.rs

```diff
@@ -133,10 +133,8 @@ pub enum PublishSendError {
     #[error("Permission denied (status code {0}): {1}")]
     PermissionDenied(StatusCode, String),
     /// See inline comment.
-    #[error(
-        "The request was redirected, but redirects are not allowed when publishing, please use the canonical URL: `{0}`"
-    )]
-    RedirectError(Url),
+    #[error("Too many redirects: {0} (max number of redirects: {1})")]
```

Comment:
In this error message, isn't the number of retry always the maximum number of redirects? i.e., can we say something like "Too many redirects, only {n_past_redirections} redirects are allowed"?